test/testQMLFile.js

```diff
@@ -1503,4 +1503,24 @@ module.exports.qmlfile = {
         test.equal(set.size(), 3);
         test.done();
     },
+    testQMLFileParseCommentedWindowStyle: function(test) {
+        test.expect(2);
+
+        var qf = new QMLFile({
+            project: p,
+            pathName: undefined,
+            type: qmlft
+        });
+        test.ok(qf);
+
+        qf.parse('        \n'+
+        '    // qsTr("Another day")\n' +
+        '    // qsTr("Another day (1)")\r\n' +
+        '    // qsTr("Another day (2)")\r' +
```

Comment:
This does not test i18n comments, only that non-i18n comments are removed. Maybe add one more test to see that i18n comments are preserved and other non-i18n comments are removed.